cmd/init.go

```diff
@@ -101,16 +100,14 @@ func configFileTemplate(tools []tools.Tool) string {
 	pmdVersion := "6.55.0"
 
 	for _, tool := range tools {
-		if tool.Uuid == "f8b29663-2cb2-498d-b923-a10c6a8c05cd" {
+		switch tool.Uuid {
+		case string(ESLint):
```

Comment:
Can we have this as a ESLintUUID or similar?
```suggestion
		case ESLintUUID:
```